app.ts

```diff
@@ -0,0 +1,271 @@
+let fromParameter = 0;
+let timer: number;
+let recordCount: number;
+
+//// On Window Load
+
+window.onload = function () {
+  getRecordCount()
+    .then(() => {
+      return getHeadings();
+    })
+    .then(() => {
+      return getTable();
+    });
+  console.log("CALL HECTOR!!!");
+};
+
+const getParameters = (fromParameter: number): number => {
+  let toParameter: number;
+  let noOfRows = getNoOfRows();
+
+  toParameter = fromParameter + noOfRows;
+  return toParameter;
+};
+
+const getNoOfRows = (): number => {
+  const height = window.innerHeight;
+  let noOfRows = Math.floor(height / 40);
+  return noOfRows;
+};
+
+//// Functions To Create/Clear The HTML
+
+// Heading Row
+
+const createHeadingRow = (headingData: string) => {
+  const heading: any = document.querySelector("#heading");
```

Comment:
The only place you can have an `any` in the project is when handling data that comes from the back-end. And even then you can actually use the real type instead of `any`.